orttraining/orttraining/python/training/ort_triton/triton_op_executor.py

```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+import functools
+import json
+import os
+import sys
+from types import ModuleType
+from typing import List, Tuple
+
+import onnx
+from torch._C import _from_dlpack
+from torch.utils.dlpack import to_dlpack
+
+from ._cache import ModuleCache, PyCodeCache
+from ._codegen import codegen
+from ._op_config import get_supported_ops
+from ._sorted_graph import SortedGraph
+from ._sympy_utils import parse_shape
+from ._utils import gen_unique_name
+
+_DEBUG_MODE = "ORTMODULE_TRITON_DEBUG" in os.environ and int(os.getenv("ORTMODULE_TRITON_DEBUG")) == 1
```

Comment:
add all the relevant env variables to https://github.com/microsoft/onnxruntime/blob/main/docs/ORTModule_Training_Guidelines.md and add some usage guidelines.